test/commands/init.test.js

```diff
@@ -164,7 +164,7 @@ describe("init - plugin devDependency", function() {
         // verify there is only one plugin with our name
         assert.strictEqual(1, config.plugins.reduce((count, plugin) => plugin.options.name === "@adobe/aio-cli-plugin-asset-compute" ? count + 1 : count, 0));
 
-    }).timeout(30000); // because of the npm install
+    }).timeout(60000); // because of the npm install
```

Comment:
Maybe drop the comments, and set the value as a constant with a clear name? Wdyt?